src/components/Main/DiscordInvite/DiscordInvite.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
-
+import {Row, Col} from 'react-bootstrap';
+// import './discordstyle.css';
```

Comment:
for TODO and commented-out files, go ahead and just delete them along with the original file